tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -366,38 +365,6 @@ describe(__filename, () => {
     );
   });
 
-  it('should dispatch a fetch action if addonType is updated', () => {
-    const { store } = dispatchClientMetadata();
-    const dispatchSpy = sinon.spy(store, 'dispatch');
-    const errorHandler = createStubErrorHandler();
-    const numberOfAddons = 6;
-
-    const root = render({
-      addonType: ADDON_TYPE_EXTENSION,
-      authorIds: [randomAuthorId2],
-      errorHandler,
-      numberOfAddons,
-      store,
-    });
-
-    dispatchSpy.resetHistory();
-
-    root.setProps({
-      addonType: ADDON_TYPE_OPENSEARCH,
-      authorIds: [randomAuthorId2],
-    });
-
-    sinon.assert.calledWith(
-      dispatchSpy,
-      fetchAddonsByAuthors({
-        addonType: ADDON_TYPE_OPENSEARCH,
-        authorIds: [randomAuthorId2],
-        errorHandlerId: errorHandler.id,
-        pageSize: String(numberOfAddons),
-      }),
-    );
-  });
```

Comment:
Could you maybe not delete this test case but replace `ADDON_TYPE_OPENSEARCH` with `ADDON_TYPE_STATIC_THEME` for example?

Reply:
Yes. Ofcourse.